modules/recommended-install-practices.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-install-practices.adoc
+
+[id="recommended-install-practices_{context}"]
+= Recommended practices for installing large scale clusters
+
+When installing large clusters or scaling the cluster to larger node counts,
```

Comment:
We should note what *large* is. I would recommend adding:

"larger node counts (anything above 1000 nodes)"